.github/workflows/v10-ci.yml

```diff
@@ -74,7 +74,7 @@ jobs:
       - name: Install dependencies
         run: yarn install --immutable --immutable-cache
       - name: Build project
-        run: yarn build --ignore '@carbon/sketch'
```

Comment:
this is a v10 workflow file, the sketch files will still exist in v10 branch even after this PR is merged in. Should we remove this configuration after removing these files from v10 (if we're going to do that...) as to not break this workflow?

Reply:
Oh, I forgot to check where this CI file runs (branch, tag, etc).

https://github.com/carbon-design-system/carbon/blob/8c21b1998401096ff59d3ab0fc31355c1f7f27c2/.github/workflows/v10-ci.yml#L2-L8

This should be reverted. I don't think it's going to break the CI because, as you said, the sketch package still exists in the v10 branch.

same with `v10-release.yml`

Reply:
This PR can focus on `main`/v11, but I think we want to also remove the plugin in the `v10` branch as well. @guidari Could you open a PR for the `v10` branch?

Reply:
Sure! I'll open a PR for the v10